### PR TITLE
An update to the README and Thank Watchdog

### PR DESCRIPTION
**Pull request by ChachyGit**
An update to the `README.MD` to make it more compilant with the discord rules.

### CODE REVIEW

File: src/main/java/cc/hyperium/integrations/watchdog/ThankWatchdog.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import cc.hyperium.config.Settings;
+import cc.hyperium.event.ChatEvent;
+import cc.hyperium.event.InvokeEvent;
+import net.minecraft.client.Minecraft;
+
+public class ThankWatchdog {
+
+  @InvokeEvent
+  public void onChat(ChatEvent e) {
+    String watchdogBanMessage = "A player has been removed from your game for hacking or abuse. Thanks for reporting it!";
```
Comment (FalseHonesty) [inline src/main/java/cc/hyperium/integrations/watchdog/ThankWatchdog.java]: Why recreate this every time? Definitely just make this final and class-level.

Comment (ChachyGit) [reply src/main/java/cc/hyperium/integrations/watchdog/ThankWatchdog.java]: so just public final void?

Comment (FalseHonesty) [reply src/main/java/cc/hyperium/integrations/watchdog/ThankWatchdog.java]: I meant the ban message

Comment (ChachyGit) [reply src/main/java/cc/hyperium/integrations/watchdog/ThankWatchdog.java]: oh, ok.

**Comment by ChachyGit** [comment]
finally

**Pull request merged**